bin/catchup_list.csv

```diff
@@ -0,0 +1 @@
+20200124
```

Comment:
I'm not sure if this was intentional or not, but let's *not* put the catchup list in bin/, and not in github at all, since we want to be able to edit it on the fly without changing the installed version.  It could be useful to have a command line option to override the default "catchup_list.csv" name.

Good that you have it setup so that if catchup_list.csv doesn't exist it just moves on with and empty list instead of crashing.